packages/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -9,7 +9,7 @@ import { Box } from '../Box/Box'
 
 export type TooltipProps = Partial<TooltipState> &
 	React.PropsWithChildren<{
-		trigger: React.ReactElement
+		trigger: React.ReactNode
```

Comment:
Had to change this since I'm using `React.PropsWithChildren` (see `WithDevToolsTooltip`) which returns a `React.ReactNode` instead of `React.ReactElement`